kodilla-good-patterns/src/main/java/MainRunner.java

```diff
@@ -0,0 +1,21 @@
+import com.kodilla.good.patterns.challenges.MovieStore;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+
+public class MainRunner {
+
+    public static void main(String[] args){
+
+       Map<String, List<String>> listMap = new MovieStore().getMovies();
+
+       String result = listMap.values().stream()
+               .flatMap(s -> s.stream())
+               .collect(Collectors.joining(" ! "," "," "));
```

Comment:
co to za dziwne argumenty? nie wystarczy `Collectors.joining(SEPARATOR)`? No i dobra praktyka: używamy stałych w takich sytuacjach, bo jak ktoś zeche nagle zmienić wykrzyknik na średnik, to musimy wszędzie, gdzie tak zrobiliśmy, zmienić.